src/components/delegate/authorize-claimer-form.tsx

```diff
@@ -50,8 +57,17 @@ export const AuthorizeClaimerForm = ({ authorizedClaimerQueryKey }: AuthorizeCla
     }
   }, [authorizedClaimerQueryKey, isConfirmed, qc])
 
+  useEffect(() => {
+    if (isConfirmed && authorizedClaimerAddress) {
+      toast({
+        title: `You've delegated your RVR balance to ${formatAddress(authorizedClaimerAddress)}`,
```

Comment:
Shouldn't the text here be inverted with line 58 text ?

Reply:
Nice catch!